app/main/views/index.py

```diff
@@ -5,6 +5,10 @@
 from app.main import main
 from app.utils import assess_contact_type
 
+@main.route('/_status')
+def healthcheck():
+    return "ok", 200
```

Comment:
For consistency with the other apps I'd go with:

```suggestion
    return jsonify(status="ok"), 200
```

Also, should this have a test?

Reply:
figured i'd keep this plaintext since the other parts of this app just return html. added test tho 👍